### ISSUE TITLE
@extend in extended class or placeholder, doesn't produce extended selector.

### ISSUE THREAD

**Issue by dimitrov-adrian** [leafo/scssphp#567]
In next example %corner placeholder is not assigned to .header-menu but only to .inline-menu

```sass

%corners {
  border-radius: 5px;
}

.inline-menu {
  > .menu-item {
    display: inline-block;
    > .sub-menu {
      display: none;
    }
    &:hover > .sub-menu {
      display: block;
      @extend %corners;
    }
  }
}

.header-menu {
  @extend .inline-menu;
}
```

Produces:
```css
.inline-menu > .menu-item:hover > .sub-menu {
  border-radius: 5px;
}
.inline-menu > .menu-item, .header-menu > .menu-item {
  display: inline-block;
}
.inline-menu > .menu-item > .sub-menu, .header-menu > .menu-item > .sub-menu {
  display: none;
}
.inline-menu > .menu-item:hover > .sub-menu, .header-menu > .menu-item:hover > .sub-menu {
  display: block;
}
```

Same code on sass (SassMeister) produce

```css
.inline-menu > .menu-item:hover > .sub-menu, .header-menu > .menu-item:hover > .sub-menu {
  border-radius: 5px;
}

.inline-menu > .menu-item, .header-menu > .menu-item {
  display: inline-block;
}
.inline-menu > .menu-item > .sub-menu, .header-menu > .menu-item > .sub-menu {
  display: none;
}
.inline-menu > .menu-item:hover > .sub-menu, .header-menu > .menu-item:hover > .sub-menu {
  display: block;
}
```

Same problem with placeholders and classes. Also if try to do with next code, it also produce nothing.
```sass
.inline-menu > .menu-item:hover > .sub-menu {
  @extend %corners;
}
```

**Comment by Cerdic** [comment]
Interesting to notice that the simple following case works fine:
```
%corners {
  border-radius: 5px;
}
.inline-menu {
    &:hover  {
      @extend %corners;
    }
}

.header-menu {
  @extend .inline-menu;
}
```
producing the right
```
.inline-menu:hover, .header-menu:hover {
  border-radius: 5px; }
```

whereas just changing one selector it's broken:
```
%corners {
  border-radius: 5px;
}
.inline-menu {
    &:hover > .sub-menu {
      @extend %corners;
    }
}

.header-menu {
  @extend .inline-menu;
}
```
gives the wrong
```
.inline-menu:hover > .sub-menu {
  border-radius: 5px; }
```

**Comment by Cerdic** [comment]
Fix with #681
